### PR TITLE
changes in dependencies.rst to resolve python version related issue

### PR DESCRIPTION
**Pull request by skc222**
Updated dependencies.rst file to resolve issue #1375

### CODE REVIEW

File: DEPENDENCIES.rst [hunk 1]
```diff
@@ -165,6 +165,14 @@ To determine which version of Python is already installed on your computer, run:
 
    python -V
 
+By default ubuntu keeps pointing at python2.7. To get ubuntu to use, by default, the installed python3, use update-alternatives commands as follows:
```
Comment (scopatz) [inline DEPENDENCIES.rst]: Please keep lines at < 80 characters

**Comment by scopatz** [comment]
Thanks for putting this in @skc222!  Also, this has conflicts, so please resolve those.

File: DEPENDENCIES.rst [hunk 2]
```diff
@@ -165,6 +165,15 @@ To determine which version of Python is already installed on your computer, run:
 
    python -V
 
+
+By default ubuntu keeps pointing at python2.7. To change this default to python3, please use:
```
Comment (katyhuff) [inline DEPENDENCIES.rst]: This looks great! I want to suggest that we include a little more detail


```
Despite having installed python3, Ubuntu installations may still point at python2.7 by default. 
If python -V returns a version of python that is not preferred, the python version can be 
changed system-wide with the update-alternatives command. 

First, you can list alternatives with the following command:

 .. code-block:: bash

   update-alternatives --list python

Ubuntu may not list any alternatives, even though you have a few versions of python installed on your system. You can make Ubuntu aware of python 2.7 and python 3.5, for example, with the following commands:

 .. code-block:: bash

   sudo update-alternatives --install /usr/bin/python python /usr/bin/python2.7 1
   sudo update-alternatives --install /usr/bin/python python /usr/bin/python3.5 2

From now on, you can switch versions with the following command, which will allow you to make a choice between them:


 .. code-block:: bash

   sudo update-alternatives --config python

```

**Comment by scopatz** [comment]
Also @skc222 please ping here when you think it is ready for another look.

**Comment by skc222** [comment]
@scopatz  @katyhuff  It is ready for another look.

**Comment by scopatz** [comment]
LGTM, thanks!

**Pull request merged**